org.ekstep.uploadcontent-1.5/editor/uploadapp.js

```diff
@@ -173,6 +178,23 @@ angular.module('org.ekstep.uploadcontent-1.5', []).controller('uploadController'
             $scope.showLoader(false);
             return;
         }
+        if($scope.validateEmptyFile($scope.uploader.getFile(0).name)) {
+            ecEditor.dispatchEvent("org.ekstep.toaster:error", {
+                message: 'File name cannot be empty',
+                position: 'topCenter',
+                icon: 'fa fa-warning'
+            });
+            $scope.showLoader(false);
+            return;
+        }else if(($scope.uploader.getFile(0).name).length > 256) {
+            ecEditor.dispatchEvent("org.ekstep.toaster:error", {
```

Comment:
we can make it parametized and pass the messages to this function. and all th validations can be handled in a comoon funciton